mautrix/bridge/user.py

```diff
@@ -51,6 +51,8 @@ class BaseUser(ABC):
     command_status: dict[str, Any] | None
     _metric_value: dict[Gauge, bool]
     _prev_bridge_status: BridgeState | None
+    _bridge_state_queue: list[BridgeState]
+    _bridge_state_loop: asyncio.Task[None] | None
```

Comment:
I don't think `asyncio.Task` takes parameters for type hinting

```suggestion
    _bridge_state_loop: asyncio.Task | None
```